apps/prairielearn/elements/pl-matching/pl-matching.css

```diff
@@ -47,3 +47,38 @@
   background-color: rgba(0, 0, 0, 0.03);
   margin-right: 0.5rem;
 }
+
+.pl-matching-container-right {
+  display: flex;
```

Comment:
This is fine, but if you add this you might want to remove the same option in line 2.